packages/spanner/system-test/spanner.js

```diff
@@ -2716,7 +2716,7 @@ var spanner = new Spanner(env);
           transaction.run('SELECT * FROM TxnTable', function(err, rows) {
             assert.ifError(err);
 
-            assert.strictEqual(rows.length, 1);
+            assert.strictEqual(rows.length, 2);
```

Comment:
This just seems like the correct value should be `2`. The transaction is run with a timestamp of the second insertion, so I would assume getting 2 rows back is correct:

- insert row with timestamp: 101
- insert row with timestamp: 102
- start transaction with timestamp: 102
- read all rows
- get 2 rows back